README.md

```diff
@@ -1,13 +1,14 @@
 # codacy-usage-report
 
-codacy-usage-report is a script that generates a CSV file with the following activity information for each Codacy user:
+codacy-usage-report is a script that generates a CSV file with the following activity information for each Codacy committer:
```

Comment:
I'm thinking if "contributor" would be a word more suited than "committer"? :thinking:

```suggestion
codacy-usage-report is a script that generates a CSV file with the following activity information for each contributor that has their commits analyzed on Codacy:
```

This concept is mentioned a lot on the GitHub and GitLab documentation (but I couldn't find many instances on the Bitbucket docs):

* https://docs.github.com/en/free-pro-team@latest/github/visualizing-repository-data-with-graphs/viewing-a-projects-contributors
* https://docs.gitlab.com/ee/development/contributing/

Reply:
I like it, and it is a term that is already used!